src/encoded/upgrade/library.py

```diff
@@ -12,3 +12,22 @@ def upgrade_library_1_2(
     if analyte:
         value["analytes"] = [analyte]
         del value["analyte"]
+
+
+@upgrade_step("library", "2", "3")
+def upgrade_library_2_3(
+    value: Dict[str, Any], system: Dict[str, Any]
+) -> Dict[str, Any]:
+    """Delete fragment_maximimum_length, fragment_mean_length, 
+    fragment_minimum_length, fragment_standard_deviation_length,
+    insert_coefficient_of_variation, insert_maximum_length,
+    insert_mean_length, insert_minimum_length, insert_standard_deviation_length,
+    target_insert_maximum_length, target_insert_mean_length, target_insert_maximum_length."""
```

Comment:
Consider removing repetition of property names here in the docstring. Also, check out PEP8 recommendations for docstring formatting.